python/amici/ode_export.py

```diff
@@ -3344,3 +3351,54 @@ def smart_subs_dict(sym: sp.Expr,
         for eid, expr in subs.items()
         if eid in sym.free_symbols
     })
+
+
+# https://gist.github.com/rectangletangle/0a0d5a2e84dd3178d348
```

Comment:
Do github gists come with a default license? :thinking:

Reply:
Yes, and its no reuse 🤦 

https://stackoverflow.com/questions/61026720/the-default-license-of-a-gist

Unlikely to hold up though due to triviality as well as prior art.

Reply:
> triviality

True... but maybe then directly leave the reference out